roles/ovirt-cluster-upgrade/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
-stop_pinned_to_host_vms: false
+# stop_non_migratable_vms is alias for stop_pinned_to_host_vms
+stop_pinned_to_host_vms: "{{ stop_non_migratable_vms | default(false) }}"
```

Comment:
If I'm reading the comments on README.rd correctly, then stop_non_migratable_vms is the 'new' name, with stop_pinned_to_host_vms being deprecated.

As such, shouldn't this be the other way around?

`stop_non_migratable_vms: "{{ stop_pinned_to_host_vms | default(false) }}"`

Effectively meaning that the only reference to stop_pinned_to_host_vms should now be here.

Reply:
None of them is deprecated, those are just aliases, we don't plan to remove any of those and both should work. But yes, Martin already mentioned that and I switched that, so removing the `stop_pinned_to_host_vms` variable in future will be simpler. Thanks a lot for reviewing.